pysal/esda/moran.py

```diff
@@ -129,6 +129,16 @@ class Moran(object):
     >>> mi_1.p_norm
     5.7916539074498452e-05
 
+    Example from http://www.lpc.uottawa.ca/publications/moransi/moran.htm
```

Comment:
This is good that we test it here, but iirc, we don't have doctests running on travis? 

Is this something we want to turn on?

Reply:
I guess since it's got both a unittest & a doctest, only running the unittest is fine. Disregard.